src/api/api.controller.ts

```diff
@@ -21,6 +21,7 @@ import { ConfigResolverService } from './config.resolver.service';
 import { FusionauthService } from './fusionauth/fusionauth.service';
 import { OtpService } from './otp/otp.service';
 import { SMSResponse } from './sms/sms.interface';
+import { RefreshRequest } from '@fusionauth/typescript-client/build/src/FusionAuthClient';
```

Comment:
You should not expose fusionauth interfaces here. This is a leaky abstraction. Let's try to have the boundaries defined for this service and the FA service.

Reply:
https://www.joelonsoftware.com/2002/11/11/the-law-of-leaky-abstractions/

Reply:
Have made improvements in this [commit](https://github.com/Samagra-Development/user-service/pull/31/commits/7c4105763cbe1736e8b9701ad4b1f2f484afd7af). Please check.